hr_course/models/hr_course.py

```diff
@@ -180,6 +183,38 @@ def cancel_course(self):
         for record in self:
             record.write(record._cancel_course_values())
 
+    @api.model
+    def send_email(self, partner):
+        email = partner.email
```

Comment:
```suggestion
        email = partner.email
```
you will have a list here